### PR TITLE
adding Izzi's updated activity

### PR DESCRIPTION
**Pull request by iHinks**

### CODE REVIEW

File: activities/config_hinks.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "tasks": [
+      {
+      "layers": [
+        [
+          "d.rast", "map=flow"
+        ]
+      ],
```
Comment (petrasovaa) [inline activities/config_hinks.json]: Looks like ponds are created but not added here to be visualized?

File: activities/hinks.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+
+def run_ponds(scanned_elev, env, **kwargs):
+    repeat = 20
```
Comment (petrasovaa) [inline activities/hinks.py]: this is too many times, 2 or 3 times should be fine here

File: activities/config_hinks.json (hunk 1 above)
Comment (iHinks) [reply activities/config_hinks.json]: Thank you, Anna! I added ponds to the list of layers with "map=ponds".

File: activities/hinks.py (hunk 2 above)
Comment (iHinks) [reply activities/hinks.py]: Thank you, that is very helpful feedback. I changed the repeat variable to be 3 in the updated pull request.

**Pull request merged**